web-app/js/portal/search/FreeTextSearchPanel.js

```diff
@@ -116,6 +114,14 @@ Portal.search.FreeTextSearchPanel = Ext.extend(Ext.Panel, {
         }
     },
 
+    onSearchSubmit: function() {
```

Comment:
ohh excellent. If the user is clicking enter, then search field won't be "dirty" and therefore it's a magnifying glass click.